LayoutTests/imported/w3c/web-platform-tests/webcodecs/videoFrame-copyTo.any-expected.txt

```diff
@@ -1,17 +1,18 @@
 
 PASS Test closed frame.
-FAIL Test copying I420 frame to a non-shared ArrayBuffer promise_test: Unhandled rejection with value: object "TypeError: Unable to copy data"
-FAIL Test copying I420 frame to a non-shared ArrayBufferView promise_test: Unhandled rejection with value: object "TypeError: Unable to copy data"
-FAIL Test RGBA frame. assert_object_equals: plane 0 layout property "stride" expected 8 got 0
+PASS Test copying I420 frame to a non-shared ArrayBuffer
+PASS Test copying I420 frame to a non-shared ArrayBufferView
+PASS Test RGBA frame.
+PASS Test NV12 frame.
 PASS Test undersized buffer.
 PASS Test incorrect plane count.
-FAIL Test stride and offset work. promise_test: Unhandled rejection with value: object "TypeError: Unable to copy data"
-FAIL Test stride and offset with padding. promise_test: Unhandled rejection with value: object "TypeError: Unable to copy data"
+PASS Test stride and offset work.
+PASS Test stride and offset with padding.
 PASS Test invalid stride.
 FAIL Test address overflow. assert_throws_js: function "() => frame.allocationSize(options)" did not throw
-FAIL Test codedRect. promise_test: Unhandled rejection with value: object "TypeError: Unable to copy data"
+PASS Test codedRect.
 PASS Test empty rect.
 PASS Test unaligned rect.
-FAIL Test left crop. promise_test: Unhandled rejection with value: object "TypeError: Unable to copy data"
+FAIL Test left crop. assert_equals: buffer contents at index 0 expected 3 but got 1
```

Comment:
Why does this one still fail?

Reply:
Only full frame copies are supported right now.
A follow-up should fix this.

Reply:
Filed https://bugs.webkit.org/show_bug.cgi?id=246393